lib/config/config-ops.js

```diff
@@ -159,7 +159,7 @@ module.exports = {
                     if (isRule) {
                         dst[i] = e;
                     } else {
-                        dst[i] = deepmerge(target[i], e, combine, isRule);
+                        dst[i] = ConfigOps.merge(target[i], e, combine, isRule);
```

Comment:
Did you avoid using `this` here to avoid confusion with mistaking what `this` refers to in a static context? Just curious more than anything else.